src/components/slices/faculty-grid.module.scss

```diff
@@ -1,107 +1,171 @@
 @import '../../base/module';
+
 .FacultyGrid {
-    position: relative;
-    padding-top: 0;
+  position: relative;
+  padding-top: 0;
+  background-color: white;
 }
-.grid {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(100%, 1fr));
-    column-gap: 30px;
-    row-gap: 60px;
 
-    @media (min-width: $tablet) {
-        grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
-    }
+.photoContainer {
+  display: flex;
+  gap: 30px;
+  flex-direction: column;
+  position: relative;
+  top: -165px;
+  @media (min-width: $tablet) {
+    top: -285px;
+  }
 }
-.name {
-    font-weight: 600;
-    font-size: 25px;
-    line-height: 38px;
-    display: block;
-    margin-bottom: 16px;
+
+.headImage {
+  width: 100%;
+  height: auto;
+  border: 5px solid white;
```

Comment:
I think this should be 10 instead of 5 for the border-width same as the employee photos.